rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -1086,6 +1089,56 @@ def testWithdrawThenPair(self):
     self.assertEquals('NO_USE', ps_1['withdrawalStatus'])
     self.assertEquals(TIME_2.isoformat(), ps_1.get('withdrawalTime'))
 
+  def test_member_ordered_stored_times(self):
+    questionnaire_id = self.create_questionnaire('questionnaire3.json')
+    questionnaire_id_1 = self.create_questionnaire('all_consents_questionnaire.json')
```

Comment:
Add a new test case for enrollmentStatusMemberTime, enrollmentStatusCoreOrderedSampleTime and enrollmentStatusCoreStoredSampleTime.

Reply:
We may want to add some tests showing that these fields get cleared when physical measurements or biobank orders get canceled, or somebody withdraws EHR consent.

Reply:
Done